ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -256,16 +256,8 @@ private MutableWorldState duplicateWorldStateAtParent() {
               if (ws.isPersistable()) {
                 return ws;
               } else {
-                var wsCopy = ws.copy();
-                try {
-                  ws.close();
```

Comment:
@matkt , as discussed - this close() is unnecessary since a non-persistable worldstate does not have resources that need to be cleaned up.  Only layered worldstates are non-persistable